config/commands.js

```diff
@@ -1676,6 +1677,13 @@ var commands = exports.commands = {
 			this.sendReply("/effectiveness OR /matchup OR /eff OR /type [attack], [defender] - Provides the effectiveness of a move or type on another type or a Pokémon.");
 			this.sendReply("!effectiveness OR !matchup OR !eff OR !type [attack], [defender] - Shows everyone the effectiveness of a move or type on another type or a Pokémon.");
 		}
+		if (target === 'weakness' || target === 'weaknesses' || target === 'weak' || target === 'resist') {
+			matched = true;
+			this.sendReply("/weakness OR /weaknesses OR /weak OR /resist [pokemon] - Provides a Pokemon's resistances, weaknesses, and immunities, ignoring abilities.");
```

Comment:
We accept all aliases in `/help [alias]`, but we generally only list one. We'll list two in rare cases where we have a canonical name and a recommended short alias.

To that end, can I ask you to change this and `/effectiveness`?